### PR TITLE
DOCAPI-6697: Edit of the toValidUTF8 description

### PR DESCRIPTION
**Pull request by BayoNet**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Category (leave one):
- Documentation for #5322

### CODE REVIEW

File: docs/en/query_language/functions/string_functions.md [hunk 1]
```diff
@@ -62,7 +62,28 @@ Returns 1, if the set of bytes is valid UTF-8 encoded, otherwise 0.
...
+
+Returned value: Valid UTF-8 string.
+
+### Example
+
+```sql
+SELECT toValidUTF8('\x00\xF0\x80\x80\x80')
+```
+```text
+┌─toValidUTF8('\0����')─┐
```
Comment (danlark1) [inline docs/en/query_language/functions/string_functions.md]: The example looks a bit strange because we won't see null character in the output.

Comment (BayoNet) [reply docs/en/query_language/functions/string_functions.md]: This is the copy-paste from `clickhouse-client`, no creativity from my side :-).

Comment (danlark1) [reply docs/en/query_language/functions/string_functions.md]: Yeah, I understand, but such example can confuse people. I would show something like 

```sql
SELECT toValidUTF8('\x61\xF0\x80\x80\x80')

SELECT toValidUTF8('a����')

┌─toValidUTF8('a����')─┐
│ a�               │
└──────────────────┘
```

Comment (vitlibar) [reply docs/en/query_language/functions/string_functions.md]: I suppose we can make this example even clearer it we add one valid character to the end:

```
SELECT toValidUTF8('\x61\xF0\x80\x80\x80b')

SELECT toValidUTF8('a����b')

┌─toValidUTF8('a����b')─┐
│ a�b              │
└──────────────────┘
```

File: docs/ru/query_language/functions/string_functions.md [hunk 2]
```diff
@@ -42,7 +42,30 @@
...
+Возвращаемое значение: Корректная строка UTF-8.
+
+### Пример
+
+```sql
+SELECT toValidUTF8('\x61\xF0\x80\x80\x80')
+```
+```text
+┌─toValidUTF8('a����')─┐
+│ a�                   │
```
Comment (vitlibar) [inline docs/ru/query_language/functions/string_functions.md]: a�b

Comment (BayoNet) [reply docs/ru/query_language/functions/string_functions.md]: Yes https://github.com/yandex/ClickHouse/pull/5348/commits/fb7f6550157c937a98d62420b0bcccbc64e03cef

Comment (vitlibar) [reply docs/ru/query_language/functions/string_functions.md]: It doesn't seem you've fixed it.

**Pull request merged**